contracts/event/EventUtils.sol

```diff
@@ -3,6 +3,8 @@
 pragma solidity ^0.8.0;
 
 library EventUtils {
+    error NotFound(string key);
```

Comment:
this can be moved to Errors.sol and renamed to EventItemNotFound

having all the Errors in one contract makes it easier to parse errors when needed